mne/viz/_3d.py

```diff
@@ -353,12 +353,56 @@ def plot_trans(info, trans_fname='auto', subject=None, subjects_dir=None,
     return fig
 
 
+def _percent_to_control_points(limits, stc_data, colormap):
+    """Private helper function to convert percentiles to control points.
+
+    Note: If using 'mne_analyze', generate cmap control points for a directly
+    mirrored cmap for simplicity. I.e., no normalization to account for a
+    2-tailed mne_analyze cmap is computed.
+
+    Parameters
+    ----------
+    limits : str | 3-tuple | dict
```

Comment:
I thought that `f` stood for frequency, no? Same like `xlim`, `ylim`, `clim`, `vlim` etc.